tests/disk/test_disk_exhaustion.py

```diff
@@ -82,7 +89,7 @@ def construct_packet_and_get_params(duthost, ptfadapter, tbinfo):
 
     # use first port of first peer_ip_ifaces pair as input port
     # all ports in second peer_ip_ifaces pair will be output/forward port
-    ptf_port_idx = mg_facts["minigraph_ptf_indices"][peer_ip_ifaces_pair[0][1][0]]
+    ptf_port_idx = mg_facts["minigraph_ptf_indices"][peer_ip_ifaces_pair[0][1][0].split(".")[0]]
```

Comment:
what's the reason to changes the ptf_port_idx? Could you run test with 202305 / 202311 image and also other topos?

Reply:
Tested on 202305 on backend and non backend testbeds

![image](https://github.com/sonic-net/sonic-mgmt/assets/48968228/25f1451d-94e0-4041-aee8-8cfefea2df42)

![image](https://github.com/sonic-net/sonic-mgmt/assets/48968228/51da56a2-7ab2-450a-a3f2-e272074ef97e)